src/fundus/publishers/de/braunschweiger_zeitung.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+import re
+from typing import List, Optional, Pattern
+
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    apply_substitution_pattern_over_list,
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class BSZeitungParser(ParserProxy):
+    class V1(BaseParser):
+        _author_substitution_pattern: Pattern[str] = re.compile(r"FUNKE Mediengruppe")
+        _paragraph_selector = XPath(
+            "//div[@class='article-body']//p[not(contains(strong, 'Meistgeklickte Nachrichten "
+            "aus der Region') or contains(strong, 'Keine wichtigen News mehr verpassen') or "
+            "@rel='author' or em[@class='print'] or contains(a, 'Jetzt Angebot und Vorteile "
+            "checken') or contains(text(), 'Lesen Sie mehr Geschichten aus')  or contains("
+            "strong, 'Mehr wichtige Nachrichten aus') or contains(strong, 'Täglich wissen, "
+            "was in') or contains(strong, 'Auch interessant') or contains(strong, 'Das könnte "
+            "Sie auch interessieren') or contains(strong, 'Lesen Sie auch') or contains("
+            "strong, 'Mehr zu dem Thema') or contains(strong, 'Mehr zum Thema') or contains("
+            "strong, 'Lesen Sie dazu') or contains(strong, 'Lesen Sie hier'))]"
```

Comment:
Here are a couple of links: https://www.braunschweiger-zeitung.de/wirtschaft/article241529964/Aida-und-Co-Der-Gaza-Krieg-erreicht-die-Kreuzfahrt.html (Lesen Sie auch, mehr zum Thema) , https://www.braunschweiger-zeitung.de/article239322873/Polizei-warnt-vor-reisenden-Handwerkern-und-gibt-Tipps.html (Jetzt Angebote...), https://www.braunschweiger-zeitung.de/panorama/article238912081/archaeologie-forensik-skellet-cambridge-deutschland.html (Das könnte Sie auch interessieren, Lesen Sie auch, Mehr zum Thema), https://www.braunschweiger-zeitung.de/gifhorn/article241448648/70-jaehriger-Gifhorner-vermisst-Polizei-bittet-um-Mithilfe.html (Täglich wissen, was in), https://www.braunschweiger-zeitung.de/salzgitter/article238264259/Unfall-in-Lebenstedt-mit-mehreren-Autos-Strassensperrung.html (Weitere Nachrichten aus, Keine wichtigen News)

I'm not just filtering Das könnte, but Das könnte Sie auch interessieren. But that shows that the other formatting definitely is the better choice :)

Reply:
I think I found a better solution. It only works for the paragraph selector though, but I think that was the critical one.

Reply:
Looks much better now 👍
I think selecting by class rather then position is safer here.
`contains(@class, 'font-sans')` instead of `position()=1` ... `\p[1]`